code/modules/recycling/disposal/bin.dm

```diff
@@ -106,6 +106,7 @@
 
 /obj/machinery/disposal/proc/place_item_in_disposal(obj/item/I, mob/user)
 	I.forceMove(src)
+	I.dropped(user)
 	user.visible_message("[user.name] places \the [I] into \the [src].", "<span class='notice'>You place \the [I] into \the [src].</span>")
```

Comment:
Can you do me a favor while you're here and change `[user.name]` to just `[user]` on this line?